app/models/questionable/answer.rb

```diff
@@ -1,8 +1,10 @@
 module Questionable
   class Answer < ActiveRecord::Base
     belongs_to :user
-    belongs_to :assignment
     belongs_to :option
+    belongs_to :parent_question, class_name: 'Questionable::Question', foreign_key: 'question_id'
+
+    belongs_to :assignment
     has_one :question, through: :assignment
```

Comment:
I was planning to remove this

```
 has_one :question, through: :assignment
```
after step 2. If I do the change now then we will break bespoke for a couple of minutes until the migration and rake task are run. 

After step 2 I will switch  

```
belongs_to :parent_question, class_name: 'Questionable::Question', foreign_key: 'question_id'
```

for 

```
belongs_to :question
```

And then after step 5 

remove  belongs_to :assignment

I don't think we can allow to leave Questionable broken even if it's for a few seconds.  Sounds good or am I missing something?